root/programs/C#/Frameworks/Infrastructure/Public/Db/DamManagedOdp/DamManagedOdp47.csproj

```diff
@@ -0,0 +1,113 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@daisukenishino 

The developers might not understand the meaning of *47*.
It might be a good idea to rename this file *DamManagedOdp.net47.csproj*.

The other files that attach the word *47* are also similar.